test/input/example.js

```diff
@@ -1 +1,3 @@
+import './global.css'
```

Comment:
I think we should create a `test/input/example-with-css.js` file, so we can test things independently.

Reply:
Yup, agree, done!